dora/stress/common/src/main/java/alluxio/stress/worker/WorkerBenchParameters.java

```diff
@@ -97,6 +97,13 @@ public final class WorkerBenchParameters extends FileSystemParameters {
           + "This argument sets the size of that window.")
   public String mSliceSize = "1s";
 
+  @Parameter(names = {"--mode"},
+      description = "Decide which policy to use for file reads."
+          // + "Possible values are: [local-only, remote-only, hash]."
+          + "Possible values are: [local-only, hash]."
+          + "By default, the mode is 'hash'")
+  public String mMode = "hash";
```

Comment:
you can use a enum here and the values can then be `LOCAL_ONLY, HASH`
https://github.com/Alluxio/alluxio/blob/1a4b2a9bf11a920f1ee54e326921d176d86e898e/dora/stress/common/src/main/java/alluxio/stress/client/ClientIOParameters.java#L35

Let's only support `LOCAL_ONLY` and `HASH`. We can add a `REMOTE_ONLY` when we implement that policy.

Here the description can be
```
      description = "Specifies which worker the test process reads from."
          + "Possible values are: [" + Arrays.toString(YourEnumClass.values()) + "]"
          + "HASH - alluxio.client.file.dora.ConsistentHashPolicy"
          + "LOCAL_ONLY - alluxio.client.file.dora.LocalWorkerPolicy"
          + "The default is HASH."
```
You may replace the strings with enum/class names by code, as you see fit.

Reply:
Description should be a constant string, or it will report: Attribute value must be constant. Here I will use a string.

Reply:
cool